src/main/java/frc/robot/controllers/ShooterController.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.controllers;
+
+import edu.wpi.first.wpilibj.Encoder;
+import frc.robot.util.Context;
+import frc.robot.util.PID;
+import com.ctre.phoenix.motorcontrol.can.*;
+import com.ctre.phoenix.motorcontrol.*;
+
+public class ShooterController {
+
+    private PID velocityPID;
+
+    private double time;
+    private double lastTime;
+    private double deltaTime;
+    private double startTime;
+
+    private double desiredVelocity;
+    private double currentVelocity;
+    private double setVelocity;
+    private double setCurrent;
```

Comment:
Unclear what setCurrent means. Is that current as in present, or current as in electrical current. If it is the electrical kind it would probably be best to call it voltage as you are setting the percent voltage being sent to the motor. The load on the motor determines the current drawn.